### ISSUE TITLE
Cannot get virtuoso working (new install, Ubuntu 16.04)

### ISSUE THREAD

**Issue by TheNavigateur** [openlink/virtuoso-opensource#736]
Followed all the steps OK up to:

```
cd var/lib/virtuoso/db 
virtuoso-t -f
```

...upon which I get:

```
The program 'virtuoso-t' is currently not installed. You can install it by typing:
sudo apt install virtuoso-opensource-6.1-bin
```

`virtuoso-t` is in the appropriate `bin` folder, but I'm not expecting to have to apt install 6.1 since I'm already cloning 7.2.5-dev.

What step/steps do I need to take to make this work as indicated?

**Comment by TallTed** [comment]
I'm betting your `PATH` environment variable doesn't include `/var/lib/virtuoso/bin` (which I believe is what you meant by "the appropriate `bin` folder").

Thus you need to execute one of these --

    cd /var/lib/virtuoso/db 
    /var/lib/virtuoso/bin/virtuoso-t -f

-- or -- 

    cd /var/lib/virtuoso/db 
    ../bin/virtuoso-t -f

-- or -- 

    cd /var/lib/virtuoso/db 
    export PATH=/var/lib/virtuoso/bin:$PATH
    virtuoso-t -f

**Comment by TheNavigateur** [comment]
OK. Just curious, does the make process normally add it to the path automatically? I (had to) ran make as sudo, and it still didn't seem to happen. Any clues why it might not have?

**Comment by TallTed** [comment]
No, the `make` process does not normally change the `PATH` variable, neither typically nor in our particular case.

Many tools do use `make install` to put their executable into a directory which is already in `PATH` (and their libraries into a directory which is already in `LD_LIBRARY_PATH`); we do not do this, for a number of reasons.

**Comment by TheNavigateur** [comment]
OK! It'd be great to update the README accordingly, as my build felt broken, even though it probably isn't.

**Comment by TheNavigateur** [comment]
In case anyone's reading this, my root directory is `/usr/local/virtuoso-opensource`, so the commands are as indicated [here](https://stackoverflow.com/a/7221514/280177).
